include/seqan3/range/container/concatenated_sequences.hpp

```diff
@@ -1329,7 +1326,7 @@ class concatenated_sequences
     //!\brief Checks whether `*this` is less than or equal to `rhs`.
     constexpr bool operator<=(concatenated_sequences const & rhs) const noexcept
     {
-        return raw_data() <= rhs.data();
+        return raw_data() <= rhs.raw_data();
```

Comment:
this seems unrelated to the ranges update?
Whats happening?

Reply:
Obviously @eseiler missed this if you look at the surrounding code. I am not sure why, but after my changes, this path got triggered and I got a deprecated warning. It needs be fixed in any case.